src/agent/src/pci.rs

```diff
@@ -182,6 +187,13 @@ impl FromStr for Path {
     }
 }
 
+pub fn rescan_pci_meta() -> anyhow::Result<()> {
+    match fs::write(PCI_RESCAN_FILE, PCI_RESCAN_SIGNAL) {
```

Comment:
This can be simplified by
```rust
fs::write(PCI_RESCAN_FILE, PCI_RESCAN_SIGNAL)
```
You will format the error in `rpc.rs`, so here even the `anyhow!` is not needed.

Reply:
Agreed. Though this should be ```fs::write(PCI_RESCAN_FILE, PCI_RESCAN_SIGNAL)?;```